managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -413,6 +413,27 @@ public Result configure(UUID customerUUID) {
       //  uuid.
       Cluster c = taskParams.currentClusterType.equals(ClusterType.PRIMARY) ?
         taskParams.getPrimaryCluster() : taskParams.getReadOnlyClusters().get(0);
+      UserIntent primaryIntent = c.userIntent;
+      for (Map.Entry<String, String> intent : primaryIntent.masterGFlags.entrySet()) {
```

Comment:
1. It is better to extract this functionality into a separate function instead of copying it 6 times.
2. Also it is not clear why to put the same item into the map (when key == key.trim and it is already there).
3. Have you checked this when you have more that one item in the map and you are updating a value in the middle of the iteration? I suppose it should fail with an error (not sure about the latest Java versions but some time ago you was unable to modify collections while iterating it).

Reply:
@SergeyPotachev 
Yeah agree, thought of doing the same (1). So moved the PR to draft state. I Will update the heading.
@jaydeepkumara please take care of the changes with test cases.